### PR TITLE
Fixes for TFT Gizmo example for CPX

### PR DESCRIPTION
**Pull request by makermelissa**
Same issue with SPIFlash and forgot to add TFT SPI selection the first time.

### CODE REVIEW

File: examples/TFTGizmo/TFTGizmo.ino [hunk 1]
```diff
@@ -25,17 +25,25 @@
...
-  #if (SPI_INTERFACES_COUNT == 1)
+  #if (SPI_INTERFACES_COUNT == 1 || defined(ADAFRUIT_CIRCUITPLAYGROUND_M0))
     Adafruit_FlashTransport_SPI flashTransport(SS, &SPI);
   #else
     Adafruit_FlashTransport_SPI flashTransport(SS1, &SPI1);
   #endif
 #endif
+
+// TFT SPI interface selection
+#if (SPI_INTERFACES_COUNT == 1)
```
Comment (ladyada) [inline examples/TFTGizmo/TFTGizmo.ino]: || defined(ADAFRUIT_CIRCUITPLAYGROUND_M0))

Comment (ladyada) [reply examples/TFTGizmo/TFTGizmo.ino]: (right?)

Comment (makermelissa) [reply examples/TFTGizmo/TFTGizmo.ino]: No, for TFT SPI, it's SPI1 on the CPX.

Comment (makermelissa) [reply examples/TFTGizmo/TFTGizmo.ino]: I forgot to mention, that I tested it. :)

Comment (ladyada) [reply examples/TFTGizmo/TFTGizmo.ino]: > oh right ya! ok

**Pull request merged**